### ISSUE TITLE
Remove <p class="hidden"> from the interactive examples

### ISSUE THREAD

**Issue by peterbe** [mdn/yari#3563]
I don't know why we have this:
https://github.com/mdn/yari/blob/b7e3ea7ed56f490c49a390fd99dda7e3c758501f/kumascript/macros/EmbedInteractiveExample.ejs#L25
Consider that `<p class="hidden">` it'll be hidden for everyone, both mobile and for assisted technologies. 

If we really need to inject a hidden message why not use an HTML comment instead?

**Comment by peterbe** [comment]
@wbamberg (CC @schalkneethling ) do you know if we can just delete this line from the macro?
Or, if not, what's the argument for keeping it? If there is an argument I can copy that in as a code comment for the next time someone files an issue just like this.

**Comment by wbamberg** [comment]
See also:

* [[Markdown] Decide what to do about class="hidden"](https://github.com/mdn/content/issues/3694)
* [Remove hidden divs from JavaScript docs](https://github.com/mdn/content/pull/3755)

**Comment by peterbe** [comment]
All `div.hidden` are always automatically deleted at build-time.
By this code: https://github.com/mdn/yari/blob/b7e3ea7ed56f490c49a390fd99dda7e3c758501f/build/index.js#L430-L434
That's the same as doing what you did in https://github.com/mdn/content/pull/3755/files but at build-time every time. 

But the `EmbedInteractiveExample.ejs` is a `p.hidden` and not a `div.hidden`. That's why it doesn't get automatically deleted. 
The whole raison-d'etre for the `div.hidden` was to help the live-sample builder in Kumascript do it's thing but once that's done, it's no point keeping those `div` tags around in the final HTML we ship. 

So I think the question remains; can do add `p.hidden` to that post-KS-build cleanup? Or can we simply remove it from the `.ejs` code?

**Comment by peterbe** [comment]
Now it's coming back to me!
We *used* to have `<div class="hidden">The source for this interactive example is stored in a GitHub` in the content itself. Repeated roughly 371 times according to https://github.com/mdn/content/pull/3755/files
So we moved it into the macro code itself here: https://github.com/mdn/yari/pull/1258 at the same time as adding that `$('div.hidden').remove()` at build-time. 

That just changed it from `<div class="hidden">bla bla interactive examples` to `<p class="hidden">bla bla interactive examples` without taking a stand as to whether it's valuable to have. 
So same thing, different ways of doing it. Essentially fixing the DRY violation in the mdn/content repo. 

Basically, that was just rearranging/removing it from the source. 
**Now, it's phase two: Can we entirely delete it from the output too?**

**Comment by peterbe** [comment]
Also, I don't think this has anything to do with the Chrome attribution hidden HTML or the `<div class="hidden">` which is used for marking out what's needed for the building of live samples.

**Comment by wbamberg** [comment]
The original purpose of this "The source for this interactive example" stuff was to be guidance for editors, back in Kuma days.

The thinking was like this: if an editor saw a problem with an interactive example, and hit "Edit" in the wiki, they'd just see the interactive example macro call:

> The margin-left CSS property sets the margin area on the left side of an element. A positive value places it farther from its neighbors, while a negative value places it closer.
>
> {{EmbedInteractiveExample("pages/css/margin-left.html")}
>
> The vertical margins of two adjacent boxes may fuse. This is called margin collapsing.

...and they would be confused. So, we'd add a little note that's visible to in the WYSIWYG editor, but not shown to users:

> The margin-left CSS property sets the margin area on the left side of an element. A positive value places it farther from its neighbors, while a negative value places it closer.
>
> {{EmbedInteractiveExample("pages/css/margin-left.html")}
>
> The source for this interactive example blah blah blah
>
> The vertical margins of two adjacent boxes may fuse. This is called margin collapsing.

That's why it wasn't added in the macro - because if it was, then an editor clicking "Edit" in the Wiki wouldn't see it, they're just see the macro call. I can't think of any reason at all to add it in a macro call.

Anyway, the upshot of https://github.com/mdn/content/issues/3694, it seems to me, is that we no longer think people will be confused by this, so we don't need any hidden text (or HTML comments) pointing editors at the interactive-examples repo any more.
